res/layout/edit_box_dialog.xml

```diff
@@ -18,19 +18,17 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:gravity="clip_horizontal"
-    android:orientation="vertical" >
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:gravity="clip_horizontal"
+              android:orientation="vertical">
```

Comment:
according to the diff above the only change within the xml file is the padding of four lines. So there is no change except for file formatting so @tobiasKaminsky proposes to remove this change.